docs/storage.md

```diff
@@ -461,40 +461,66 @@ For the full discussion about Ceph in XCP-ng, see this forum thread: <https://xc
 
 ## ISO SR
 
-You might wonder: how to upload an ISO? Unlike other solutions, you need to create a dedicated "space" for those, a specific ISO SR. To create one, it's pretty easy.
+You might wonder: how to upload an ISO? Unlike other solutions, you need to create a dedicated "space" for those, a specific ISO SR. To create a ISO SR, you have 2 possibilities:
+- Shared: A shared ISO SR is on a VM or in your dedicated storage server. It's accessible with a IP address, like 192.168.1.100
+- Local (not recommended): Local ISO SR is a directory created directly on the dom0. It's only accessible on the pool where the dom0 ith the directory on it.
 
-In [Xen Orchestra](management.md#xen-orchestra) go into "New/Storage" and select "ISO SR":
+Create a Shared ISO SR:
 
-![](https://xcp-ng.org/assets/img/screenshots/createSRISO.png)
+First, you need to create the NFS or SMB Share. The creation depends on the device you want to use to store your ISO, but here a small tutorial to mount a NFS share on Debian 11:
 
-Usually, you can use a NFS or SMB share. But if you don't have any of those, you can rely on a local ISO SR that you create yourself:
+```
+fdisk /dev/xvdb #create a new partition
+mkfs.ext4 -b 4096 /dev/xvdb1 # format the partition to ext4
+apt install nfs-kernel-server
+mkdir /ISOs
+mount /dev/xvdb1 /ISOs #mount the disks to /ISOs
+emacs /etc/fstab
+  /dev/xvdb	  /ISOs	  /ext4	  defaults	0	1
+emacs /etc/exports
+  /ISOs 192.168.X.0/24(rw,sync,no_subtree_check,no_root_squash)
+exportfs -arv
+```
 
-1. Select "Local" instead of NFS/SMB
-2. Enter a path that already exists on your physical host
-3. Upload ISO on your host to the same path
+Then, in Xen Orchestra go into "New/Storage" and select "ISO SR":
 
-That's it!
+![](https://xcp-ng.org/assets/img/screenshots/createSRISO.png)
+
+Create a Local ISO SR:
 
 From the CLI:
 
-1. Create a directory on the local filesystem to storage your ISOs
-2. Copy/move ISOs to this new location
-3. Create the ISO SR using xe sr-create
-4. You can add or update ISOs later by placing them into the directory you created in step 1
-5. Rescan the SR if you change the files stored in the ISO directory
+    Create a directory on the local filesystem to storage your ISOs
+    Copy/move ISOs to this new location
+    Create the ISO SR using xe sr-create
+    You can add or update ISOs later by placing them into the directory you created in step 1
+    Rescan the SR if you change the files stored in the ISO directory
```

Comment:
You made it a code block. I don't think it's a formatting improvement
![image](https://user-images.githubusercontent.com/451043/186887765-deeef919-cf10-4472-8056-a8c3372d78ad.png)

Reply:
Yes, code block is a bad idea (i didn't know it will be displayed like that). Maybe dotted list ?

Reply:
Yes a dotted list would fit. Or leave the numbered list as before.
![image](https://user-images.githubusercontent.com/451043/187639491-91d5e4e3-1996-4336-9ee2-214cdefe83a4.png)

Reply:
Ok, changed to numbered list